index.html

```diff
@@ -14,13 +14,17 @@
     <div class="container">
         <header class="header-page">
             <!--header contents goes here-->
-
             <nav>
                 <!--This is my Logo-->
-                <a href=''>Solorics</a>
-
+                <a class="logo" href=''>Solorics</a>
+                <ul class="nav-links">
+                    <li><a href="#portfolio">Portfolio</a></li>
+                    <li><a href="#contacts">Contaacts</a></li>
+                    <li><a href="about">About</a></li>
+                    <li><a href="#mailbox"><img src="./Images/Union.png" alt="mail_box"></a></li>
+                </ul>
                 <!--Toggle button goes here-->
-                <img src="images/Union.png" alt="Toggle_Image_Button" class="toggle">
+                <!--<img src="images/Union.png" alt="Toggle_Image_Button" class="toggle">-->
```

Comment:
- [ ] Great job implementing the changes requested by the previous reviewer, images are working fine now, but the desktop version or your new changes generally shouldn't affect the mobile design 
in the mobile menu, the nav links should disappear and the hamburger button shows up 


![image](https://user-images.githubusercontent.com/89282221/188207453-afe3ef46-3e0a-41cb-99fe-58542aa53bd0.png)